addons/mail/views/res_partner_views.xml

```diff
@@ -7,16 +7,18 @@
             <field name="model">res.partner</field>
             <field name="inherit_id" ref="base.view_partner_form"/>
             <field name="arch" type="xml">
-                <xpath expr="//field[@name='email']" position="replace">
+                <xpath expr="//div[@id='base_email']" position="attributes">
+                    <attribute name="class">o_row o_row_readonly</attribute>
```

Comment:
While we are at it, can't it be set directly in base view ? What is the purpose of this code actually ?